hedera-node/src/main/java/com/hedera/services/exceptions/ValidationUtils.java

```diff
@@ -49,6 +52,28 @@ public static void validateFalse(final boolean flag, final ResponseCodeEnum code
 		}
 	}
 
+	/**
+	 * <p>Validates a particular condition.</p>
+	 * <p>If the condition is true - logs the error and throws an {@link InvalidTransactionException}.</p>
+	 * @param flag
+	 * 		the condition, which, if true, triggers logging and throwing the exception with the provided response code
+	 * @param code
+	 * 		the response code passed as an argument to the exception
+	 * @param logger
+	 * 		functional interface, which logs the error message
+	 * @param errorMessage
+	 * 	    the error message, build where the method is being called
+	 */
+	public static void validateFalse(final boolean flag,
```

Comment:
I think that the log message is an outdated log added due to concerns with the signature validation mechanism. There are plenty of use-cases where keys are used in the transition logic and there is no logging. My suggestion is to remove this version of `validateFasle` and not log the error